ddtrace/sampler.py

```diff
@@ -276,25 +278,26 @@ def sample(self, span):
                     self._set_priority(span, AUTO_REJECT)
                     return False
 
-            # If no rules match, use our default sampler
-            matching_rule = self.default_sampler
+            # If no rules match, use our default rule sampler
```

Comment:
Can we nest this into an `else` on the above branch? IMO would make it a bit more readable and possibly avoid a bug if the branches are changed to no longer `return`

Reply:
yeah, I almost did that, I'll change.